File: 8-kyu/How good are you really.py
# DESCRIPTION:
# There was a test in your class and you passed it. Congratulations!
# But you're an ambitious person. You want to know if you're better than the average student in your class.
#
# You receive an array with your peers' test scores. Now calculate the average and compare your score!
#
# Return True if you're better, else False!
#
# Note:
# Your points are not included in the array of your class's points. For calculating the average point you may add your point to
# the given array!
def better_than_average(class_points, your_points):
    count_person = 0
    class_p = 0
    for i in class_points:
        class_p+=i
        count_person+=1
    if ((class_p/count_person)>your_points):
        return False
    else: return True
# print(better_than_average([12, 23, 34, 45, 56, 67, 78, 89, 90], 69))

File: JavaScript/8-kyu/How good are you really.js
// Description:
// There was a test in your class and you passed it. Congratulations!

// But you're an ambitious person. You want to know if you're better than the average student in your class.

// You receive an array with your peers' test scores. Now calculate the average and compare your score!

// Return true if you're better, else false!

// Note:
// Your points are not included in the array of your class's points. Do not forget them when calculating the average score!

function betterThanAverage(classPoints, yourPoints) {
    let count_person = 0
    let class_p = 0
    classPoints.forEach(element => {
        class_p+=element;
        count_person+=1;
    });
    if ((class_p/count_person)>yourPoints){
        return false
    }
    else return true
  }
  